hedging/src/adjustment_action.rs

```diff
@@ -164,10 +164,97 @@ mod tests {
         let liability = dec!(100);
         let exposure = dec!(100);
         let adjustment = determine_action(liability, exposure);
-        println!("The adjustment suggested: {adjustment:?}");
         assert_eq!(
             adjustment,
             AdjustmentAction::Sell(BtcUsdSwapContracts::from(2))
         );
     }
+
+    #[test]
+    fn low_bound_limit() {
+        // condition: exposure / liability == LOW_BOUND_RATIO_SHORTING
+        // expected:  do nothing
+        let nominal_liability = 10000;
+        let liability = Decimal::from(nominal_liability);
+        let exposure = Decimal::from(-nominal_liability) * LOW_BOUND_RATIO_SHORTING;
+
+        let adjustment = determine_action(liability, exposure);
+        assert_eq!(adjustment, AdjustmentAction::DoNothing);
+    }
+
+    #[test]
+    fn low_bound_below() {
+        // condition: exposure / liability < LOW_BOUND_RATIO_SHORTING
```

Comment:
It seems like these tests are simply mirroring the implementation... hmm.

Also I'm not a fan of the comments. I see their value but perhaps they should at least be moved above the fn as doc comments:
https://doc.rust-lang.org/rust-by-example/meta/doc.html
```
/// ...
#[test]
fn ..
```

Reply:
I can do the calculations offline and put magic numbers here, it will be the same "mirroring" of the implementation, no choice else they won't pass. So those unit test will protect from implementation changes breaking them.

Comments were for me before writing the code, I can remove them, we don't have any, anywhere.

What do you think?

Reply:
If you don't need them anymore my preference is to delete the comments.

no magic numbers - I guess the mirroring is okay for now - can't think of anything better.

Reply:
perfect will do that. thx